massgen/frontend/displays/textual_widgets/content_sections.py

```
@@ -942,11 +1000,22 @@ def add_tool(self, tool_data: ToolDisplayData, round_number: int = 1) -> ToolCal
         # Close any open reasoning batch when tool arrives
         self._close_reasoning_batch()
 
+        # Debug logging - include widget ID to identify which panel
+        widget_id = self.id or "unknown"
+        with open("/tmp/tui_debug.log", "a") as f:
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/tui_debug.log"

(S108)